uui-core/src/types/tables.ts

```diff
@@ -251,6 +255,9 @@ export interface DataTableCellProps<TItem = any, TId = any, TCellValue = any> ex
 
     /** Overrides default tooltip, used to show validation message if the cell is invalid */
     renderTooltip?: (props: ICanBeInvalid & TooltipCoreProps) => React.ReactElement;
+
+    /** Defines columns gap size */
+    columnsGap?: '12' | '24';
```

Comment:
I think, `columnsGap` is a global table prop, we shouldn't give a opportunity to define gap for each columns separately, so we can remove this prop from this interface.

Reply:
done